### PR TITLE
Releasenotes v8.14: Credits for doc added

### PR DESCRIPTION
**Pull request by StephanStS**

### CODE REVIEW

**Comment by MichaIng** [comment]
Would this fit better below a "Documentation" header instead of below "Enhancements"? A short sentence about what has been changed (per commit/PR) could then be added as well.

**Comment by StephanStS** [comment]
> Would this fit better below a "Documentation" header instead of below "Enhancements"? A short sentence about what has been changed (per commit/PR) could then be added as well.

Thought about this, but propose not to bloat the releasenotes with these informations.
What I could imagine would be a short list like e.g.

```
- @JappeHallunken, @sheepy0125, @hello-jason: Typo
- @danielpinto8zz6: Orange Pi 5 docs
```

**Comment by MichaIng** [comment]
> What I could imagine would be a short list like e.g.

Sounds good. About bloating, I think if we already add credits, then a little hint about what has been done is fine. I mean its release notes and not release credits 😉.

**Review by MichaIng — APPROVED** [review]

File: docs/releases/v8_14.md [hunk 1]
```diff
@@ -34,6 +34,11 @@ The **February 11th, 2023** release of **DietPi v8.14** comes with a new image f
...
 - [**DietPi-Software**](../../dietpi_tools/software_installation/#dietpi-software) | [**Docker Compose**](../../software/programming/#docker-compose) :octicons-arrow-right-16: Migrated to the plugin variant, installed from the Docker APT repository. On existing installs, the migration can be done via `dietpi-software reinstall 134`. To call it, use `docker compose` (without dash) from now on. Many thanks to @isarrider for making us aware of this: <https://github.com/MichaIng/DietPi/issues/6135>
 - **DietPi-Globals** :octicons-arrow-right-16: G_CHECK_NET: When hitting "Retry" from the error handler, the IPv6 connection check is now skipped as well when IPv6 is disabled via `dietpi-config` network settings or manually. The same is true for changes to the `CONFIG_CHECK_CONNECTION/DOMAIN` `dietpi.txt` settings, which can now also be changed right from the error handler. Many thanks to @derSebastian for reporting a related issue: <https://github.com/MichaIng/DietPi/issues/5667>
+- **DietPi-Docs** :octicons-arrow-right-16: Documentation contributions were done from a couple of persons.  
+Many thanks to
```
Comment (MichaIng) [inline docs/releases/v8_14.md:38]:
```suggestion
Many thanks to
```

**Pull request merged**